app/views/questions/_question.html.erb

```diff
@@ -2,15 +2,15 @@
 <table>
   <tr>
     <td>
-      <% if logged_in? && current_user.posted_question?(question) %>
+      <% if logged_in? %>
```

Comment:
I noticed you wrote a few user helper methods, which look great! If you reference the users_controller:line 13, I'm setting a session there upon successful user registration.  We should decide if we all want to use the logged_in helper method instead, so that we don't accidentally set too many sessions in the code.  Let me know if that doesn't make sense.